charts/ChartUrl.ts

```diff
@@ -234,16 +234,20 @@ export class ChartUrl implements ObservableUrl {
         }
 
         const time = params.time
-        if (time !== undefined) {
+        if (time) {
```

Comment:
This won't work for `"0"`.

Reply:
I think it will? `""` is falsy, but `"0"` is truthy.

Reply:
You're right 😆